content/docs/next-release/deployment.md

```diff
@@ -2,6 +2,9 @@
 title: Deployment
 weight: 7
 children:
+- title: Operator
+  url: operator
+children:
```

Comment:
`children` is repeated, making the second override the first. It should be:

```yaml
children:
- title: Frontend/UI
  url: frontend-ui
- title: Operator
  url: operator
```

Which then renders this:
![image](https://user-images.githubusercontent.com/13387/58793355-36d6b300-85f6-11e9-9ca1-c0d6407586d0.png)